File: codingbat/java/sum67.java
public int sum67(int[] nums) {
  boolean stopSum = false;
  int total = 0;
  for (int num: nums) {
    if (num == 6) stopSum = true;
    if (!stopSum) total += num;
    if (num == 7) stopSum = false;
  }
  return total;
}

File: codingbat/python/sum67.py
def sum67(nums):
  is_skip = False
  total = 0
  for num in nums:
    if not is_skip and num == 6:
      is_skip = True
      continue
    if is_skip and num == 7:
      is_skip = False
      continue
    if not is_skip: total += num
  return total

